src/service/fixed-layer.js

```diff
@@ -295,6 +295,17 @@ export class FixedLayer {
   removeFixedElement_(element) {
     for (let i = 0; i < this.fixedElements_.length; i++) {
       if (this.fixedElements_[i].element == element) {
+        this.vsync_.run({
```

Comment:
As discussed, just `element.style.top = ''` should be enough, but we may also need `element.style.zIndex = ''` - please check.